__tests__/todo.spec.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { render, fireEvent, cleanup } from '@testing-library/react-native';
+import { build, fake } from '@jackfranklin/test-data-bot';
+import { firebase as mockFirebase } from '../src/firebase/config';
+
+import Todo from '../src/screens/todo';
+
+// mockFirebase.firestore().useEmulator('localhost', 8080);
+
+const userDataBuilder = build('User Data', {
+  fields: {
+    uid: fake((f) => f.internet.password()),
```

Comment:
`fake((f) => f.internet.password())` -> `fake((f) => f.datatype.uuid())`

Reply:
Actually that wont work, it trows `datatype` as `undefined` **BUT** `uid: fake((f) => f.random.uuid())` and **its still a uuid**

Reply:
`f.random.uuid` seems much better than `f.internet.password()` can we update?

Reply:
Updated on commit #93847f6, second to last commit